API.md

```diff
@@ -96,15 +109,23 @@ Takes a screenshot of the browser window where the specified web page is opened.
 Object that contains information about the browser installed on the machine.
 
 **Kind**: global typedef  
+**Note**: If BrowserInfo#winOpenCmdTemplate is defined on Windows, it will be rendered and used as open command.
```

Comment:
Sorry, but API.md was outdated a bit, because I've forgot to run `docs` task to regenerate it. Maybe we should add this task to dependencies of `build-lib`?

Sharp `#` notes that `winOpenCmdTemplate` is instance member of `BrowserInfo` (dot `.` is used for statics, and tilde `~` - for internals). But I agree, that it looks too heavy, especially in plain text, because jsdoc doesn't generate links to class fields. It was simplified to `winOpenCmdTemplate` in source comments, but API.md is outdated.

On Mac we always need macOpenCmdTemplate, because we don't have any other way to build execution string on Mac: https://github.com/AndreyBelym/testcafe-browser-natives/blob/master/src/api/open.js#L33